packages/core-data/src/resolvers.js

```diff
@@ -165,8 +171,29 @@ export const getEntityRecord =
 					}
 				}
 
-				const record = await apiFetch( { path } );
-				dispatch.receiveEntityRecords( kind, name, record, query );
+				const response = await apiFetch( { path, parse: false } );
```

Comment:
Setting `parse: false` will disable the `fetchAllMiddleware`. See how it [returns early](https://github.com/WordPress/gutenberg/blob/c4332b60ee79235e76bf6926e8368a72b62b6284/packages/api-fetch/src/middlewares/fetch-all-middleware.js#L78-L81) when `options.parse` is `false`. Is this a concern here?

`fetchAllMiddleware` is relevant for fetching collections, and `getEntityRecord` fetches only one item, so it shouldn't be relevant. But what if I do:
```js
getEntityRecord( 'postType', 'post', '', { per_page: -1 } );
```
This will fetch the entire collection anyway, won't it? And should trigger `fetchAllMiddleware`.